src/app/components/datatable-issuer-detail.component.ts

```diff
@@ -0,0 +1,107 @@
+import { BadgeInstance } from '../issuer/models/badgeinstance.model';
+import { FormsModule } from '@angular/forms';
+import { HlmIconComponent, provideIcons } from '../../../components/ui-icon-helm/src';
+import { lucideSearch } from '@ng-icons/lucide';
+import { HlmCommandInputWrapperComponent } from '../../../components/ui-command-helm/src';
+
+
+@Component({
+	selector: 'issuer-detail-datatable',
+	standalone: true,
+	imports: [
+    FormsModule,
+		HlmTableModule,
+		HlmIconModule,
+		CommonModule,
+		BadgrCommonModule,
+		TranslateModule,
+		BrnAccordionContentComponent,
+		RouterModule,
+    HlmInputDirective,
+    HlmLabelDirective,
+    HlmIconComponent,
+    HlmCommandInputWrapperComponent
+    ],
+    providers: [provideIcons({ lucideSearch })],    
+	template: `
+  <div class="tw-p-[calc(var(--gridspacing)*2)] tw-mt-8 tw-max-w-[596px]">
+    <div class="tw-flex tw-items-center tw-justify-between tw-gap-4 sm:flex-col">
+      <div class="l-stack u-margin-bottom2x u-margin-top4x">
+            <h3 class="md:tw-text-xl tw-text-sm tw-font-semibold tw-font-[rubik] tw-text-oebblack">
+              {{ recipientCount }} Badge {{ recipientCount == 1 ? 'Empfänger:in' : 'Empfänger:innen' }}
+            </h3>
+      </div>
+      <label hlmLabel class="tw-font-semibold tw-text-[0.5rem] tw-w-full"> 
+        <span class="tw-px-3">Nach Email Adresse suchen</span>
+        <hlm-cmd-input-wrapper class="tw-relative tw-px-0 tw-mt-1">
+                <input
+                hlmInput
+                class="tw-w-full tw-border-solid tw-ml-auto tw-border-purple tw-py-1 tw-rounded-[20px]"
+                [ngModel]="_emailFilter()"
+                (ngModelChange)="_rawFilterInput.set($event)"
+                />
+                <hlm-icon size="lg" class="tw-absolute  tw-right-6 tw-text-purple" name="lucideSearch" />
+        </hlm-cmd-input-wrapper>
+      
+        </label>
+      </div>
+        <hlm-table class="tw-rounded-[20px] tw-overflow-hidden tw-w-full tw-max-w-[100%] tw-bg-lightpurple tw-border-purple tw-border tw-mt-8">
+            <hlm-caption>{{caption}}</hlm-caption>
+            <hlm-trow class="tw-bg-purple tw-text-white tw-flex-wrap hover:tw-bg-purple">
+                <hlm-th class="!tw-text-white tw-w-40">ID</hlm-th>
+                <hlm-th class="!tw-text-white tw-justify-center !tw-flex-1">Vergeben am </hlm-th>
+            </hlm-trow>
+            <hlm-trow *ngFor="let recipient of _filteredEmails()" class="tw-border-purple tw-flex-wrap tw-py-2">
+                <hlm-th class="tw-w-40">
+                    <span class="!tw-text-oebblack !tw-font-normal">{{recipient.recipientIdentifier}}</span>
+                </hlm-th>
+                <hlm-th class="!tw-flex-1 tw-justify-center !tw-text-oebblack"><p class="u-text"><time [date]="recipient.issuedOn" format="dd/MM/y"></time></p></hlm-th>
+
+            </hlm-trow>
+        </hlm-table>
+      </div>  
+        `,
+})
+
+
+
+export class IssuerDetailDatatableComponent {
```

Comment:
Where is this used? I couldn't find it in the frontend

Reply:
It is used in the ```badgeclass detail component``` html file, you should find it if you search by the selector ```issuer-detail-datatable```

Reply:
@jona159 I didn't mean in the HTML files but in the browser; where do I need to navigate to find this component?

Reply:
Ah sorry. Go to ```Meine Institutionen``` -> click on your institution -> click on a badge that has been awarded at least once. Then you should see it. 
Or if you prefer a link ```https://deploy-preview-335--openeducationalbadges.netlify.app/issuer/issuers/<your_issuer_id>/badges/<your_badgeclass_id>```